apps/src/studio/studio.js

```diff
@@ -5382,8 +5382,17 @@ Studio.setScoreText = function(opts) {
   Studio.displayScore();
 };
 
-Studio.setVictoryText = function(opts) {
-  Studio.victoryText = opts.text;
+/**
+ * Sets up Studio.victoryText and Studio.victoryValue
+ * Studio.victoryText is the text to display to a user when their level has met it's win or lose condition
+ * Studio.victoryValue is either 'win' or 'lose' despite the language settings for the page. This can be used to
+ *  determine if an end condition has been reached.
+ * @param {string} - text The translated string to display to the user when their game has reached an end condition.
+ * @param {string} - value Either 'win' or 'lose' depending on the end condition which was reached.
+ */
+Studio.setVictoryText = function(text, value) {
+  Studio.victoryText = text;
+  Studio.victoryValue = value;
```

Comment:
how feasible would it be for `value` to be a boolean, instead? Something like `Studio.isVictory`?

Reply:
Thanks for asking about this. I investigated further and found some boolean flags which already exist. Only the levels need to be updated in this change.